spec/spec_helper.rb

```diff
@@ -147,7 +152,7 @@ def set_hieradata(hieradata)
 Dir.glob("#{RSpec.configuration.module_path}/*").each do |dir|
   begin
     Pathname.new(dir).realpath
-  rescue
-    fail "ERROR: The module '#{dir}' is not installed. Tests cannot continue."
+  rescue StandardError
+    raise "ERROR: The module '#{dir}' is not installed. Tests cannot continue."
```

Comment:
StandardError is general enough that we should at least be able to eyeball why it failed. 

```suggestion
  rescue StandardError => e
    raise "ERROR: The module '#{dir}' is not installed (#{e.message}). Tests cannot continue."
```